torch_tvm/operators.cpp

```diff
@@ -215,6 +219,55 @@ RegisterTVMOperator reg({
        }
        return out;
      }},
+    {Symbol::fromQualString("aten::layer_norm"),
+     [](Node* node, tvm::Array<tvm::relay::Expr> inputs) -> tvm::relay::Expr {
+       auto op = tvm::relay::Op::Get("nn.custom_layer_norm");
+       TORCH_CHECK(
+           inputs.size() == 6,
+           "layer_norm received ",
+           inputs.size(),
+           " inputs");
+       auto normalized_axis_shape = relayToArray<tvm::Integer>(inputs[1]);
+       std::vector<int64_t> shape;
+       shape.reserve(normalized_axis_shape.size());
+       for (auto& dim : normalized_axis_shape) {
+         shape.push_back(dim);
+       }
+       auto attrs = tvm::make_node<tvm::relay::CustomLayerNormAttrs>();
+       attrs->num_axis_to_normalize = normalized_axis_shape.size();
+       auto eps = static_cast<double>(relayToConstant<float>(inputs[4]));
+       attrs->eps = eps;
+
+       TVMContext ctx_;
+       ctx_.device_type = kDLCPU;
```

Comment:
hmm, how will this fair in situations where we don't have CPU guaranteed?

Reply:
Do you mean what happens when CPU is not available? Or the Tensors from pytorch are not on CPU? I think that requires handling outside of all this. From the implementations of other operators it does not seem to me that we handle cases for non-CPU devices.
In fact I would think that we would need to check that the graph being lowered has inputs on CPU until we expand the scope.

Reply:
if we set the `ctx_` in the module to GPU would this still work?

Reply:
I think it should work ok. But I wouldn't trust without testing.